tests/test_pickle.py

```diff
@@ -393,25 +430,6 @@ def test_receiver(self, pickle):
         assert np.all(new_rec.data == 1)
         assert np.all(new_rec.coordinates.data == [[0.], [1.], [2.]])
 
-    def test_alias_sparse_function(self, pickle):
-        grid = Grid(shape=(3,))
-        sf = SparseFunction(name='sf', grid=grid, npoint=3, space_order=2,
-                            coordinates=[(0.,), (1.,), (2.,)])
-        sf.data[0] = 1.
-
-        # Create alias
-        f0 = sf._rebuild(name='f0', alias=True)
```

Comment:
Are we dropping the test for this alias=True case?

the new test that you've added above doesn't appear to be testing this case

Reply:
Slightly moved above